sensor-helm-chart/templates/sensor-deployment.yaml

```diff
@@ -89,6 +89,16 @@ spec:
               secretKeyRef:
                 name: {{ .Values.secretName | quote }}
                 key: SENTRY_DSN
+          - name: RADAR_THRESHOLD_MILIS
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.secretName | quote }}
+                key: RADAR_THRESHOLD_MILIS
+          - name: DOOR_THRESHOLD_MILIS
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.secretName | quote }}
+                key: DOOR_THRESHOLD_MILIS
```

Comment:
`MILIS` should have two "L"s

Reply:
yes this is what I botched when deploying this